platforms/posix/src/lockstep_scheduler/include/lockstep_scheduler/lockstep_scheduler.h

```diff
@@ -10,22 +10,40 @@
 class LockstepScheduler
 {
 public:
+	~LockstepScheduler();
+
 	void set_absolute_time(uint64_t time_us);
-	uint64_t get_absolute_time() const;
+	inline uint64_t get_absolute_time() const { return _time_us; }
```

Comment:
I like that you made it `inline` explicitly but it is already `inline` automatically if defined in the class declaration, right?

Reply:
> I like that you made it inline explicitly but it is already inline automatically if defined in the class declaration, right?

Yes, this is just slightly stronger, and sometimes I like to add it to make it clear to a reader that it's important here. Not strictly necessary though.